src/Native/Runtime/unix/unixasmmacrosamd64.inc

```diff
@@ -238,7 +238,17 @@ C_FUNC(\Name):
 #define THREAD__VALIDATEEXINFOPOP C_FUNC(_ZN6Thread17ValidateExInfoPopEP6ExInfoPv)
 
 .macro INLINE_GETTHREAD
-        call    C_FUNC(RhpGetThread)
+        // Inlined version of call C_FUNC(RhpGetThread)
+       .att_syntax
```

Comment:
What was the problem causing us to switch to the att syntax here? If it was the addq at the end, we can use lea there - the ELF TLS doc uses lea there too.